resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/RateLimiterConfig.java

```diff
@@ -117,6 +124,14 @@ private static int checkLimitForPeriod(final int limitForPeriod) {
         return limitForPeriod;
     }
 
+    private static Instant checkStartedTime(final Instant startedTime) {
+        requireNonNull(startedTime, STARTED_TIME_MUST_NOT_BE_NULL);
+        if (startedTime.isAfter(Instant.now())) {
+            throw new IllegalArgumentException("StartedTime should be earlier or equal than now");
+        }
+        return startedTime;
+    }
```

Comment:
FYI) if we allow to set `startTime` that later than now, we need to fix lots of codes on `AtomicRateLimiter`.

https://github.com/resilience4j/resilience4j/blob/24798b3aa800be28e28207da33f717583179be82/resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/internal/AtomicRateLimiter.java#L66
https://github.com/resilience4j/resilience4j/blob/24798b3aa800be28e28207da33f717583179be82/resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/internal/AtomicRateLimiter.java#L103-L108

Cause `AtomicRateLimiter` assumes that `startTime` is the time when class is loaded and always `now() > startTime`

https://github.com/resilience4j/resilience4j/blob/24798b3aa800be28e28207da33f717583179be82/resilience4j-ratelimiter/src/main/java/io/github/resilience4j/ratelimiter/internal/AtomicRateLimiter.java#L222-L223

And it calculate `currentCycle` based on `currentNanoTime()` (elapsed time after class loaded), so if we set `future` as start time here then `currentCycle` is negative and limiter not works well.

Also, I think it's unclear what limiter should do when `now()` is before `startTime`.

So I named it as `startedTime` and want to set instant that earlier or equal than now~!